modules/ocf_irc/files/znc.service

```diff
@@ -0,0 +1,11 @@
+[Service]
+ExecStart=/usr/bin/znc --foreground --datadir /opt/znc
+User=znc
+Group=znc
+Restart=always
+StandardOutput=syslog
+StandardError=syslog
+SyslogIdentifier=znc
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Alright, well this works anyway, I imagine either way will work fine anyway, it's not like there are that many reloads anyway.